rfcs/text/0000-dss-metrics.md

```diff
@@ -0,0 +1,65 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC: Key performance metrics of the DCP Data Store (DSS)
+
+## Summary
+
+This RFC proposes a set of key performance metrics to be introduced and tracked as a summary measure of performance by
+which the utility of the Data Store can be evaluated, including improvements and regressions.
+
+## Author(s)
+
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Beyond latency and error rates, the DSS lacks a clear focus on quantifiable assessment of usability. We would like to
+quantify higher level performance metrics that reflect the Data Store's ability to make HCA data accessible, computable,
+and durable.
+
+### User Stories
+
+* As a DSS developer, I would like to know whether the DSS is improving in its ability to serve downstream users and
+  services.
+
+* As a downstream service developer, I would like to suggest a way of measuring DSS performance that reflects my ability
```

Comment:
I like this user story a lot but I don't see any mention in the detailed design of how the metrics provided allows for flexibility to suggest new metrics.

Perhaps this user story is more intended for another RFC around how to request for new metrics instead of this RFC which enumerates the initial set of metrics.